apps/cowswap-frontend/src/modules/account/containers/Transaction/StatusDetails.tsx

```diff
@@ -97,8 +98,11 @@ export function StatusDetails(props: StatusDetailsProps) {
         isCancelling={isCancelling}
         isPresignaturePending={isPresignaturePending}
         isCreating={isCreating}
+        title={isReplaced ? 'Transaction was cancelled or speed up' : ''}
```

Comment:
This is in the past, so it should be either `sped up` or `speeded up` https://grammarwiki.com/past-tense-of-speed/

I prefer `sped up`.

Reply:
Thank you! Fixed